### PR TITLE
Kafka connect benchmark

### PR DESCRIPTION
**Pull request by okkez**

### CODE REVIEW

**Review by cosmo0920 — APPROVED** [review]
LGTM!

File: kafka-connect/FluentdSourceConnector.properties [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name=FluentdSourceConnector
+tasks.max=1
+connector.class=org.fluentd.kafka.FluentdSourceConnector
+fluentd.port=24224
+fluentd.bind=0.0.0.0
+fluentd.transport=tls
```
Comment (cosmo0920) [inline kafka-connect/FluentdSourceConnector.properties]: Ah, wait...should we use tls transport when non-tls version of benchmark?

Comment (okkez) [reply kafka-connect/FluentdSourceConnector.properties]: Removed.

Comment (cosmo0920) [reply kafka-connect/FluentdSourceConnector.properties]: Thanks!

**Pull request merged**